internal/testing/cluster_test.go

```diff
@@ -2174,22 +2203,38 @@ func TestSlabBufferStats(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// assert number of objects
 	os, err = b.ObjectsStats()
 	if err != nil {
 		t.Fatal(err)
 	}
 	if os.NumObjects != 2 {
-		t.Fatal("expected 1 object, got", os.NumObjects)
-	}
-	if os.TotalObjectsSize != uint64(len(data1)+len(data2)) {
-		t.Fatalf("expected totalObjectSize of %v, got %v", len(data1)+len(data2), os.TotalObjectsSize)
-	}
-	if os.TotalSectorsSize != 3*rhpv2.SectorSize {
-		t.Fatalf("expected totalSectorSize of %v, got %v", 3*rhpv2.SectorSize, os.TotalSectorsSize)
+		t.Fatalf("expected 1 object, got %d", os.NumObjects)
 	}
-	if os.TotalUploadedSize != 3*rhpv2.SectorSize {
-		t.Fatalf("expected totalUploadedSize of %v, got %v", 3*rhpv2.SectorSize, os.TotalUploadedSize)
+
+	// check the object size stats, we use a retry loop since packed slabs are
+	// uploaded in a separate goroutine, so the object stats might lag a bit
+	err = Retry(60, time.Second, func() error {
```

Comment:
Seems like you are not spawning a goroutine in the upload (anymore?)

Reply:
thanks, I removed/added it to see if tests were passing and forgot to re-add it 🙄